Community/AssetManagement/lumeta_workflow_page.py

```diff
@@ -0,0 +1,311 @@
+# Copyright 2019 BlueCat Networks. All rights reserved.
+
+import ipaddress
+from flask import request, g, abort, jsonify
+from bluecat.api_exception import PortalException, APIException
+from bluecat import route, util
+from main_app import app
+
+# application config
+
+# Define global variable to hold handle to API object
+api = None
+
+#
+# GET, PUT or POST
+#
+@route(app, '/lumeta/getnetworklist', methods=['GET', 'PUT', 'POST'])
+@util.rest_workflow_permission_required('lumeta_workflow_page')
+@util.rest_exception_catcher
+def get_networks_get_networks_page():
+    # are we authenticated?
+    g.user.logger.info('SUCCESS')
+    configurations = None
+    configurations_json = []
+    if g.user:
+        configurations = g.user.get_api().get_configurations()
+        for c in configurations:
+            print (c)
+            configuration_json = {"id": c.get_id(), "name": c.get_name()}
+            configurations_json.append(configuration_json)
+        return jsonify(configurations_json)
+
+
+@route(app, '/lumeta/getiplist', methods=['GET', 'PUT', 'POST'])
```

Comment:
For this endpoint a few of the comments above apple as well, I won't repeat them.